amethyst_renderer/src/formats/texture.rs

```diff
@@ -57,6 +108,12 @@ pub enum TextureData {
     U64(Vec<u64>, Option<TextureMetadata>),
 }
 
+impl From<[f32; 4]> for TextureData {
+    fn from(color: [f32; 4]) -> Self {
+        TextureData::Rgba(color, None)
```

Comment:
Maybe we could not wrap metadata in `Option` anymore?

Reply:
Possibly, I thought about it. I'll remove the Option, users can just use Default instead